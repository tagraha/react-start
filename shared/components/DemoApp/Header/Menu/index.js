import React from 'react';
import Link from 'react-router-dom/Link';

function Menu() {
  return (
    <ul style={{ marginTop: '1rem', padding: '1rem', backgroundColor: '#c6c6c6' }}>
      <li><Link to="/">Home</Link></li>
      <li><Link to="/counter">Counter</Link></li>
      <li><Link to="/about">About</Link></li>
    </ul>
  );
}

export default Menu;
