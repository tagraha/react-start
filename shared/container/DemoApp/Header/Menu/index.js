/* @flow */
/* eslint-disable jsx-a11y/anchor-is-valid */

import React from 'react';
import Link from 'react-router-dom/Link';

function Menu() {
  return (
    <ul
      style={{
        marginTop: '1rem',
        padding: '1rem',
        border: '1px solid #c6c6c6',
      }}
    >
      <li>
        <Link to="/">Home</Link>
      </li>
      <li>
        <Link to="/counter">Counter</Link>
      </li>
      <li>
        <Link to="/async">Async example</Link>
      </li>
      <li>
        <Link to="/about">About</Link>
      </li>
    </ul>
  );
}

export default Menu;
