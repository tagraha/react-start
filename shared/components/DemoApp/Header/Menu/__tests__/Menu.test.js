/* eslint-disable import/no-extraneous-dependencies */

import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

import Menu from '../index';

describe('<Home />', () => {
  test('renders', () => {
    const wrapper = shallow(<Menu />);
    expect(wrapper).toMatchSnapshot();
  });
});
