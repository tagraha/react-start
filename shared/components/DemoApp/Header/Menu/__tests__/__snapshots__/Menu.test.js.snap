// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders 1`] = `
<ul
  style={
    Object {
      "backgroundColor": "#c6c6c6",
      "marginTop": "1rem",
      "padding": "1rem",
    }
  }
>
  <li>
    <Link
      replace={false}
      to="/"
    >
      Home
    </Link>
  </li>
  <li>
    <Link
      replace={false}
      to="/counter"
    >
      Counter
    </Link>
  </li>
  <li>
    <Link
      replace={false}
      to="/about"
    >
      About
    </Link>
  </li>
</ul>
`;
